### DISCUSSION TITLE
template multiple if wont work together inside a template

### DISCUSSION THREAD

**Discussion by RohitM-IN** [alpinejs/alpine#2246] [1. Help] [answered]
i tried running similar code in my app but its not working is there any workaround because the inside template changes too much and you cant use `x-show` it needs Boolean 
 
https://codepen.io/rootandroid58/pen/MWvjbjQ

it seems `x-if` breaks out of loop when first if fails  hence its not even going to second template  or even third :( 

is there any workaround for this

**Comment by SimoTod** [comment, marked as answer]
No workarounds, just follow the docs. ;) Alpine templates must have a single child item, if you wrap your 3 sub templates in a single div, it works

**Comment by Kawkawa2** [comment]
i have same problem here, am trying to display certain content if the condition matches but it checks only the first condition then displays everything inside the template, any suggestions?

**Reply by SimoTod** [reply]
Did the answer above not work for you?

**Reply by ekwoka** [reply]
No way to help without seeing code.
